Source/API/MultiSearch.lua

```diff
@@ -0,0 +1,31 @@
+local function InitializeAPIMultiSearchFrame()
+  if Auctionator.State.APIMultiSearchFrameRef == nil then
+    local frame = CreateFrame(
+      "FRAME",
+      "AuctionatorAPIMultiSearchFrame",
+      AuctionHouseFrame,
+      "AuctionatorAPIMultiSearchFrameTemplate"
+    )
+
+    Auctionator.State.AuctionAPIMultiSearchFrameRef = frame
+  end
+end
+
+function Auctionator.API.v1.MultiSearch(callerID, searchTerms)
+  Auctionator.API.InternalVerifyID(callerID)
+
+  --Validate arguments
+  local cloned = Auctionator.Utilities.VerifyListTypes(searchTerms, "string")
+  if not cloned then
+    Auctionator.API.ComposeError(
+      callerID, "Usage Auctionator.API.v1.MultiSearch(string[])"
+    )
+  end
+  -- Validate state
+  if not AuctionHouseFrame or not AuctionHouseFrame:IsShown() then
```

Comment:
Should we do this check before bothering to assemble the list?

Reply:
I just went with verify parameters, then verify state

Reply:
But I don't think there's any benefit to doing it earlier, either way they have to fix both problems

Reply:
Makes sense.